cmd/clusterctl/test/README.md

```diff
@@ -0,0 +1,17 @@
+# Running the tests
+
+	./run-e2e.sh
```

Comment:
I assume the hack should be called before run-e2e test, or I'm wrong?

Reply:
Yes, you're right. I have updated the docs. The docker override folder will have to be deleted since we generated it's yamls in the test script. 
We can add auto-generation of all other component yamls is in a follow up PR.